source/authentication/api-key.txt

```diff
@@ -81,123 +67,168 @@ Configuration
            }
          }
 
-.. note::
+.. _api-key-user-objects:
+
+API Key User Objects
+--------------------
+
+Every App Services user has a unique metadata object. The object is passed to Functions
+called by the user and rule expressions for requests made by the user.
+In API key user objects, the ``type`` field has the value ``"server"``. For example:
+
+.. code-block:: js
+   :emphasize-lines: 3
 
-   The API Key authentication provider does not have any
-   provider-specific configuration options.
+   {
+     id: "<Unique User ID>",
+     type: "server",
+     data: <user data object>,
+     custom_data: <custom user data object>,
+     identities: <array of user identities>,
+   }
+
+You can use this field to evaluate if requests come from API keys.
+
+For more information, refer to :ref:`User Objects <user-objects>`.
+
+.. _api-key-authentication-server-api-key:
+
+Server API Keys
+---------------
+
+Server API keys are generated in a server-side context using one of
+the App Services admin interfaces. When you create a server API key,
```

Comment:
Pretty vague. Which Admin interface(s)?

Reply:
fair, that's more of an internally facing term. removing references to be more specific-- the App Services CLI, API or UI